### PR TITLE
Update sbt to 1.6.2

### PR DESCRIPTION
**Pull request by tdroxler**

### CODE REVIEW

**Comment by tdroxler** [comment]
got a flaky test:
```
MinerApiControllerSpec
should handle submission *** FAILED *** (3 seconds, 139 milliseconds)
java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsg while waiting for Subscribe
at scala.Predef$.assert(Predef.scala:279)
at akka.testkit.TestKitBase.expectMsg_internal(TestKit.scala:461)
at akka.testkit.TestKitBase.expectMsg(TestKit.scala:438)
at akka.testkit.TestKitBase.expectMsg$(TestKit.scala:438)
at akka.testkit.TestKit.expectMsg(TestKit.scala:973)
at org.alephium.flow.mining.MinerApiControllerSpec$Fixture.connectToServer(MinerApiControllerSpec.scala:50)
at org.alephium.flow.mining.MinerApiControllerSpec$Fixture.connectToServer$(MinerApiControllerSpec.scala:48)
at org.alephium.flow.mining.MinerApiControllerSpec$$anon$3.connectToServer(MinerApiControllerSpec.scala:96)
at org.alephium.flow.mining.MinerApiControllerSpec$$anon$3.<init>(MinerApiControllerSpec.scala:98)
at org.alephium.flow.mining.MinerApiControllerSpec.$anonfun$new$5(MinerApiControllerSpec.scala:96)
```

File: build.sbt [hunk 1]
```diff
@@ -19,9 +19,8 @@ lazy val root: Project = Project("alephium-scala-blockflow", file("."))
   .settings(commonSettings: _*)
   .settings(
     name := "alephium",
-    unmanagedSourceDirectories := Seq(),
```
Comment (polarker) [inline build.sbt:22]: This is not needed any more?

Comment (tdroxler) [reply build.sbt:22]: sbt said it was unused

**Review by polarker — COMMENTED** [review]
LGTM. That flaky test happens sometimes. Let's restart for now

**Pull request merged**